packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -40,4 +40,12 @@ Terra.describeViewports('StatusView', ['medium'], () => {
     browser.url('/raw/tests/terra-status-view/status-view/status-view-thin-constraint');
     Terra.validates.element('error too thin for glyph');
   });
+  it('Displays scrolled Status View', () => {
+    browser.url('/raw/tests/terra-status-view/status-view/status-view-overflow');
+    // eslint-disable-next-line prefer-arrow-callback
+    browser.execute(function scrollOptionIntoView() {
+      document.querySelector('p[class*="title"]').scrollIntoView();
+    });
+    Terra.validates.element('scrolled', { selector: '#root' });
```

Comment:
can we use the test component's id for the screenshot instead? i'd rather have a screenshot of the component itself, not the whole page

Reply:
Updated here https://github.com/cerner/terra-core/pull/3392/commits/b960ae6e215b94422d0db6eb5ec69289524df161.